packages/ts/react-form/src/index.ts

```diff
@@ -5,31 +5,32 @@ import {
   type BinderConfiguration,
   BinderRoot,
   CHANGED,
-  defaultValidity,
+  type FieldStrategy,
   getBinderNode,
   getDefaultFieldStrategy,
   hasFromString,
   isFieldElement,
   type ModelConstructor,
   type Validator,
   type ValueError,
-  type FieldStrategy,
 } from '@hilla/form';
 import type { BinderNode } from '@hilla/form/BinderNode.js';
 import { useEffect, useMemo, useReducer, useRef } from 'react';
+import type { VaadinWindow } from './types.js';
 
+declare const __VERSION__: string;
 
-const $wnd = window as any;
+const $wnd = window as VaadinWindow;
 
 $wnd.Vaadin ??= {};
 $wnd.Vaadin.registrations ??= [];
 $wnd.Vaadin.registrations.push({
   is: '@hilla/react-form',
-  version: /* updated-by-script */ '2.2.0-beta1',
+  version: __VERSION__,
```

Comment:
Is the script still able to update the value? Where did the default value go?

Reply:
The `__VERSION__` is replaced by esbuild to the version from `package.json` during the build stage. We already use this idea in `@hilla/form`. It allows us to maintain the version in a single place and avoid unnecessary version-update commits.

Reply:
Good then